### PR TITLE
Improved consistency of logging chart

### PR DESCRIPTION
**Pull request by a-thaler**
**Description**

Changes proposed in this pull request:

Consistent usage of
- naming and labeling
- file naming
- templating
Migrated log-ui sub-chart to be just a sub-folder

### CODE REVIEW

File: resources/logging/templates/loki/servicerolebinding.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+metadata:
+  name: {{ template "loki.fullname" . }}
+  labels:
+    app: {{ template "loki.name" . }}
+    chart: {{ template "loki.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  subjects:
+  - user: "cluster.local/ns/kyma-system/sa/logging-loki"
```
Comment (hisarbalik) [inline resources/logging/templates/loki/servicerolebinding.yaml]: we should also update clusterrbacconfig file with updated service name, file placed in the chart resources/core/charts/istio-rbac/templates/rbac-config.yaml

File: resources/logging/templates/loki/servicerole.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    app: {{ template "loki.name" . }}
+    chart: {{ template "loki.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  rules:
+  - services: ["logging.kyma-system.svc.cluster.local"]
+    methods: ["GET", "POST", "OPTIONS"]
+    paths: ["/api/*"]
+  - services: ["logging.kyma-system.svc.cluster.local"]
```
Comment (hisarbalik) [inline resources/logging/templates/loki/servicerole.yaml]: service name should be updated to logging-loki

File: resources/logging/templates/loki/servicerole.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+metadata:
+  name: {{ template "loki.fullname" . }}
+  labels:
+    app: {{ template "loki.name" . }}
+    chart: {{ template "loki.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  rules:
+  - services: ["logging.kyma-system.svc.cluster.local"]
```
Comment (hisarbalik) [inline resources/logging/templates/loki/servicerole.yaml]: service name should be updated to logging-loki

File: resources/logging/templates/loki/servicerolebinding.yaml (hunk 1 above)
Comment (a-thaler) [reply resources/logging/templates/loki/servicerolebinding.yaml]: done

File: resources/logging/templates/loki/servicerole.yaml (hunk 3 above)
Comment (a-thaler) [reply resources/logging/templates/loki/servicerole.yaml]: used proper template now

File: resources/logging/templates/loki/servicerole.yaml (hunk 2 above)
Comment (a-thaler) [reply resources/logging/templates/loki/servicerole.yaml]: using proper template now

**Comment by a-thaler** [comment]
@kwiatekus well spotted, applied the suggestion

**Pull request merged**